index.js

```diff
@@ -65,6 +65,17 @@ class K8sVMExecutor extends Executor {
 
             return err || !status || status.toLowerCase() === 'pending';
         };
+
+        const tolerations = hoek.reach(options, 'kubernetes.tolerations', { default: [] });
+
+        if (Array.isArray(tolerations) && tolerations.length) {
+            this.tolerations = tolerations.map(toleration => ({
+                key: toleration.key,
+                value: toleration.value,
```

Comment:
If the operator is `Exists`, there should be no `value`. Do we need to delete the `value` key in that case?
https://kubernetes.io/docs/concepts/configuration/taint-and-toleration/#concepts

Reply:
My guess is it will be ignored, rather than error out. It's value will be undefined. Will have to test this though.

Reply:
I changed input format. Now rather than `tolerations` it expects an `nodeSelectors` which is an object whose key-value represents cluster lever node selectors.